bundles/logging/log_helper/include/celix_log_helper.h

```diff
@@ -90,6 +90,14 @@ void celix_logHelper_logDetails(celix_log_helper_t* logHelper,
                                 int line,
                                 const char *format, ...) __attribute__((format(printf,6,7)));
 
+/**
+ * @brief Logs a message using the provided celix log level to the log_helper, printf style.
+ *        It will also print celix thread-specific storage error messages(celix_err).
+ *
+ * Silently ignores log level CELIX_LOG_LEVEL_DISABLED.
+ */
+void celix_logHelper_logWithTssErrors(celix_log_helper_t* logHelper, celix_log_level_e level, const char *format, ...) __attribute__((format(printf,3,4)));
```

Comment:
Is it an idea to only add a `celix_logHelper_logTssErrors(celix_log_helper_t* logHelper, celix_log_level_e level)` function and a C++ `celix::LogHelper::logTssErrors(celix_log_level_e level)`?

This requires an extra call for the users (using log helper to log a message and then log celix err errors), but prevents that we (ideally) create an alternative for the `log`, `vlog`, `logDetails` and `vlogDetails` in C and C++.